CoreScriptsRoot/Modules/Settings/Utility.lua

```diff
@@ -1546,51 +1725,101 @@ local function CreateNewSlider(numOfSteps, startStep, minStep)
 
 	leftButton.InputBegan:connect(function(inputObject) mouseDownFunc(inputObject, currentStep - 1, true) end)
 	leftButton.InputEnded:connect(function(inputObject) mouseUpFunc(inputObject) end)
-	leftButton.MouseButton1Click:connect(function()
-		if UserInputService.TouchEnabled and not UserInputService.GamepadEnabled then
-			touchClickFunc(inputObject, currentStep - 1, true)
-		end
-	end)
 	rightButton.InputBegan:connect(function(inputObject) mouseDownFunc(inputObject, currentStep + 1, true) end)
 	rightButton.InputEnded:connect(function(inputObject) mouseUpFunc(inputObject) end)
-	rightButton.MouseButton1Click:connect(function()
-		if UserInputService.TouchEnabled and not UserInputService.GamepadEnabled then
-			touchClickFunc(inputObject, currentStep + 1, true)
+
+	local function onVREnabled(prop)
+		if prop ~= "VREnabled" then
+			return
 		end
-	end)
-	
+		if UserInputService.VREnabled then
+			leftButton.Selectable = interactable
+			rightButton.Selectable = interactable
+			this.SliderFrame.Selectable = interactable
+
+			for i = 1, steps do
+				this.Steps[i].Selectable = interactable
+				this.Steps[i].Active = interactable
+			end
+		else
+			leftButton.Selectable = false
+			rightButton.Selectable = false
+			this.SliderFrame.Selectable = interactable
+			for i = 1, steps do
+				this.Steps[i].Selectable = false
+				this.Steps[i].Active = false
+			end
+		end
+	end
+	if fixSettingsMenuVR then
+		UserInputService.Changed:connect(onVREnabled)
+		onVREnabled("VREnabled")
+	end
+
 	for i = 1, steps do
-		this.Steps[i].InputBegan:connect(function(inputObject) mouseDownFunc(inputObject, i) end)
-		this.Steps[i].InputEnded:connect(function(inputObject) mouseUpFunc(inputObject) end)
+		this.Steps[i].InputBegan:connect(function(inputObject) 
+			mouseDownFunc(inputObject, i) 
+		end)
+		this.Steps[i].InputEnded:connect(function(inputObject)
+		 mouseUpFunc(inputObject) end)
 	end
 
-	this.SliderFrame.InputBegan:connect(function(inputObject) mouseDownFunc(inputObject, currentStep) end)
-	this.SliderFrame.InputEnded:connect(function(inputObject) mouseUpFunc(inputObject) end)
+	this.SliderFrame.InputBegan:connect(function(inputObject) 
+		if fixSettingsMenuVR then
```

Comment:
I think you may be right, although clicking does still work. This just breaks being able to click anywhere in the slider frame to change the value (rather than exactly on a cell.) I'll make this VR-only.